srcpkgs/cpptest/template

```diff
@@ -0,0 +1,35 @@
+# Template file for 'cpptest'
+pkgname=cpptest
+version=1.1.2
+revision=1
+build_style=gnu-configure
+configure_args="--prefix=/usr"
```

Comment:
this is redundant, gnu-configure already passes --prefix=/usr

Reply:
Fixed.